packages/at_secondary_server/lib/src/verb/handler/pkam_verb_handler.dart

```diff
@@ -116,10 +172,9 @@ class PkamVerbHandler extends AbstractVerbHandler {
     }
     logger.finer('hashingAlgoEnum: $hashingAlgoEnum');
```

Comment:
Please follow dart style and change `var hashingAlgoEnum` above to `HashingAlgoType? hashingAlgoEnum;`

Reply:
changed as part of [05f42a0](https://github.com/atsign-foundation/at_server/pull/1520/commits/05f42a080aa3b989ce7d6bddb7dbc3489b2825d5)